wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -58,9 +58,15 @@
   private static final Logger LOG = LoggerFactory.getLogger(WorkspaceManager.class);
 
   /** This attribute describes time when workspace was created. */
-  public static final String CREATED_ATTRIBUTE_NAME = "created";
-  /** This attribute describes time when workspace was last updated or started/stopped/recovered. */
-  public static final String UPDATED_ATTRIBUTE_NAME = "updated";
+  protected static final String CREATED_ATTRIBUTE_NAME = "created";
+  /** This attribute describes time when workspace was last updated or started */
+  protected static final String UPDATED_ATTRIBUTE_NAME = "updated";
+  /** This attribute describes time when workspace was last stopped */
+  protected static final String STOPPED_ATTRIBUTE_NAME = "stopped";
+  /** This attribute indicates is last stop was abnormal */
+  protected static final String STOPPED_ABNORMALLY_ATTRIBUTE_NAME = "stoppedAbnormally";
+  /** This attribute describes latest startup error message */
+  protected static final String ERROR_MESSAGE_ATTRIBUTE_NAME = "errorMessage";
```

Comment:
It would be better to rename this attribute to make it clear that it is startup error message.